src/brsTypes/components/Group.ts

```diff
@@ -0,0 +1,29 @@
+import { RoSGNode } from "./RoSGNode";
+import { ValueKind } from "../BrsType";
+import { AAMember } from "./RoAssociativeArray";
+
+export class Group extends RoSGNode {
+    readonly kind = ValueKind.Object;
```

Comment:
I don't think this line is necessary since it's provided by RoSGNode.

Reply:
Nice catch! 👍